pkg/image/controller/factory.go

```diff
@@ -63,7 +63,7 @@ func (opts ScheduledImageStreamControllerOptions) GetRateLimiter() flowcontrol.R
 // NewImageStreamController returns a new image stream import controller.
 func NewImageStreamController(client imagev1client.Interface, informer imagev1informer.ImageStreamInformer) *ImageStreamController {
 	controller := &ImageStreamController{
-		queue: workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "ImageStreamController"),
+		queue: workqueue.NewNamedRateLimitingQueue(workqueue.DefaultItemBasedRateLimiter(), "ImageStreamController"),
```

Comment:
I think this is the right change - I expected us to rate limit streams or namespaces, not globally.

Reply:
Is there a way to split workqueue by namespace effectively vs by item?